examples/stable-video-diffusion/models/svd.py

```diff
@@ -0,0 +1,88 @@
+import os
+import requests
+
+from dataclasses import dataclass
+from typing import List, Union, Iterable
+
+from PIL import Image
+import numpy as np
+import torch
+
+from nos.hub import HuggingFaceHubConfig
+
+@dataclass(frozen=True)
+class StableVideoDiffusionConfig(HuggingFaceHubConfig):
+    
+    model_cls: str = "StableVideoDiffusionPipeline"
+
```

Comment:
Add docstrings if possible for each field. See https://github.com/autonomi-ai/nos/blob/main/nos/models/stable_diffusion.py#L14 for reference.

Reply:
Done.